lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -30,6 +30,7 @@ export default (accessToken, userID, requestIDs) => {
       "is_saved"
     ),
     saleArtworksLoader: gravityLoader(id => `sale/${id}/sale_artworks`),
+    saleArtworksFilterLoader: gravityLoader("filter/sale_artworks"),
```

Comment:
Since this relies on its own endpoint independent of a `saleId` the naming of this seemed appropriate; let me know if you'd like me to revise.